### PR TITLE
View current user's user name on logout menu. Resolving Issue #125

### PR DESCRIPTION
**Pull request by KaveeshBaddage**
## Description

When system user is going to logout from the system, current user's user name should view in the Logout menu by the system.Currently  that feature is not included. This pull request add required implementation to view current user's name in the logout menu.
## Motivation and Context

This gives the opportunity to the user to know from which account he has been used to login to the system.

## How Has This Been Tested?

After implementing this feature, I have logged to the system using different user accounts and checked is it work correctly. It's worked correctly all the time.

## Screenshots :
View of the logout menu before implementing

bassalogoutview

View of the logout menu after implementing

bassalogoutmenuwithusername

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ #] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to hange)

## Checklist:

- [ #] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rajikaimal** [comment]
Codacy is complaining about not having `camel_case` notation in `MainController.js`.

**Comment by KaveeshBaddage** [comment]
In Maincontroller.js file, I have used a variable named "user_name". That variable was declared in login.html file. It was existed variable in login.html when I was  starting the implementation of this functionality and I didn't go th change that variable. Does it need to change from the beginning?

**Comment by rajikaimal** [comment]
If that's the case changing `user_name` might cause breaking changes. Better to ping @malithsen about this. :running_man:

**Comment by KaveeshBaddage** [comment]
@rajikaimal Ok cool..  @malithsen Does it need to change "user_name"  variable name from the beginning or Is it okay  to use in this form ?

**Comment by malithsen** [comment]
@KaveeshBaddage Nice work :+1: 
I'll merge the PR as is.
Please consider issuing another PR addressing the styling inconsistencies
1) `user_name` 
2) We have used both `var vm = this` and $scope. Since we have used $scope in most files, I think it's better to stay with it. So let's replace `vm` with `$scope` in `MainController.js` as well

File: ui/src/app/login/controllers/LoginCtrl.js [hunk 1]
```diff
@@ -6,11 +6,14 @@
 
   function LoginCtrl($scope, $state, UserService) {
     $scope.user = {};
+
```
Comment (malithsen) [inline ui/src/app/login/controllers/LoginCtrl.js]: Let's remove these empty lines

Comment (KaveeshBaddage) [inline ui/src/app/login/controllers/LoginCtrl.js]: Okay. I will work on that problem.

**Comment by KaveeshBaddage** [comment]
Thank you @malithsen. :smile:  I have removed empty lines from LoginCtrl.js file. I will work on those issues which you notified above.

**Comment by malithsen** [comment]
https://github.com/scorelab/Bassa/blob/9d1bdd666584e5a63461464a2528e9962974528f/ui/src/app/login/controllers/LoginCtrl.js#L13

Just noticed that this stores the password in plaintext in localstorage :scream:

**Pull request merged**